public/email.svg

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
@tillprochaska please tell me where you got this icon set from and how to add more icons.